tests/test_api.py

```diff
@@ -2050,198 +2050,296 @@ def test_simple(self):
             self.assertEqual(match["image"], None)
-         
+
     def test_limit(self):
         """
         Test limited global search
         """
         if not self.sg.server_caps.version or self.sg.server_caps.version < (6, 2, 0):
             return
-         
-        result = self.sg.text_search("%s Text Search" % self._prefix, {"Shot" : [] }, limit=3 )        
+
+        result = self.sg.text_search("%s Text Search" % self._prefix, {"Shot" : [] }, limit=3 )
         matches = result["matches"]
         self.assertEqual(len(matches), 3)
- 
+
     def test_entity_filter(self):
         """
         Test basic multi-type global search
         """
         if not self.sg.server_caps.version or self.sg.server_caps.version < (6, 2, 0):
             return
-         
-        result = self.sg.text_search("%s Text Search" % self._prefix, 
+
+        result = self.sg.text_search("%s Text Search" % self._prefix,
                                      {"Shot": [], "Asset": [] } )
-         
+
         matches = result["matches"]
-         
+
         self.assertEqual(set(["matches", "terms"]), set(result.keys()))
         self.assertEqual(len(matches), 10)
-                 
- 
+
+
     def test_complex_entity_filter(self):
         """
         Test complex multi-type global search
         """
         if not self.sg.server_caps.version or self.sg.server_caps.version < (6, 2, 0):
             return
-         
-        result = self.sg.text_search("%s Text Search" % self._prefix, 
-                                     {"Shot": [["code", "ends_with", "3"]], 
+
+        result = self.sg.text_search("%s Text Search" % self._prefix,
+                                     {"Shot": [["code", "ends_with", "3"]],
                                       "Asset": [
-                                                {"filter_operator": "any", 
+                                                {"filter_operator": "any",
                                                  "filters": [["code", "ends_with", "4"]]
                                                 }
                                                 ]
                                      })
-         
+
         matches = result["matches"]
-         
+
         self.assertEqual(set(["matches", "terms"]), set(result.keys()))
         self.assertEqual(len(matches), 2)
-         
+
         self.assertEqual(matches[0]["type"], "Shot")
         self.assertEqual(matches[0]["name"], "%s Text Search 3" % self._prefix)
         self.assertEqual(matches[1]["type"], "Asset")
         self.assertEqual(matches[1]["name"], "%s Text Search 4" % self._prefix)
-         
-        
 
 
+class TestReadAdditionalFilterPresets(base.LiveTestBase):
+    """
+    Unit tests for the additional_filter_presets read parameter
+    """
+
+    def test_simple_case(self):
+
+        filters = [
+            ['project', 'is', self.project],
+            ['id', 'is', self.version['id']]
```

Comment:
Should switch to double quotes throughout the tests (to be in line with our python coding conventions).